grandcanyon.tm2/project.xml

```diff
@@ -0,0 +1,3238 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE Map[]>
+<Map srs="+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0.0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs +over" background-color="#ffffff">
+
+<Parameters>
+  <Parameter name="attribution"><![CDATA[<a href="http://geomaps.wr.usgs.gov/arizona/">George Billingsley et al.</a> <a href="https://www.mapbox.com/about/maps/">© Mapbox © OpenStreetMap</a> and Ryan Clark, 2014]]></Parameter>
+  <Parameter name="bounds">-180,-85.0511,180,85.0511</Parameter>
+  <Parameter name="center">-111.9843,36.1498,11</Parameter>
+  <Parameter name="format">png8:m=h</Parameter>
+  <Parameter name="maxzoom">14</Parameter>
+  <Parameter name="minzoom">10</Parameter>
+  <Parameter name="name"><![CDATA[Geologic Map of the Grand Canyon]]></Parameter>
+  <Parameter name="source"><![CDATA[mapbox:///rclark.u33j714i,mapbox.mapbox-terrain-v2,mapbox.mapbox-streets-v5]]></Parameter>
```

Comment:
@brandon-rhodes This is where source data is pulled, and the rest of the document is rules for styling that source data.

```
mapbox:///rclark.u33j714i,mapbox.mapbox-terrain-v2,mapbox.mapbox-streets-v5
```

This is a set of 3 vector tile datasets. The URI here is an instruction to a Mapbox rendering service to pull data from the 3 sources, blend them into a single set of vector data, then apply the styling rules from the rest of the document.

- rclark.u33j714i: This is the geologic tileset that I produced using old software
- mapbox.mapbox-terrain-v2: This is an old version of hillshade data that's provided by Mapbox
- mapbox.mapbox-streets-v5: This is a very old version of street/trail/landuse data that's provided by Mapbox